README.md

```diff
@@ -1,13 +1,13 @@
 # Cycle Music
-![Screencapture](http://zupimages.net/up/17/45/x655.png)
+![Screencapture](https://i.goopics.net/01nxw.png)
 
 ## Sources
  - [reactive programming -medium- @tkssharma](https://medium.com/@tkssharma/reactive-programming-rx-js-introduction-32bf963eee1b)
  - [xstream README.md](https://github.com/staltz/xstream)
  - [cycle.js.org documentation](https://cycle.js.org/getting-started.html)
 
 ## Slides
- - [Web2day 2017](https://docs.google.com/presentation/d/1ffUXgs4dimSTOKi3RtfW_6FYeabH9mQs19S1Om2-73k/edit?usp=sharing)
+ - [BDX.io 2017](https://docs.google.com/presentation/d/1QipKxyUIcntcfenZPb6IzpMv11nKSCUyfVfB5VIgaWI/edit?usp=sharing)
```

Comment:
Is it the branch to merge or one living branch ?

Reply:
one living

Reply:
Ok, we should update the master to add slides too then

Reply:
And to refert to this branch, nice job 👍